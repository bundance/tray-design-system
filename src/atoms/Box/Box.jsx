import React from "react";
import { space, width, fontSize, color } from "styled-system";
import styled, { ThemeProvider } from "styled-components";
import theme from "../../theme/theme";

const StyledBox = styled.div`
  ${color}
  ${fontSize}
  ${space}
  ${width}
`;

export default () => (
  <ThemeProvider theme={theme}>
    <StyledBox p={3} bg="whites.10" color="orange">
      This is a Box
    </StyledBox>
  </ThemeProvider>
);
