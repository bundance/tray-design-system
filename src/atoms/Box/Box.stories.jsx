import React from "react";
import { storiesOf } from "@storybook/react";

import Box from "./Box";

storiesOf("atoms/Box", module).add("Default Box", () => <Box>Hello Box</Box>);
