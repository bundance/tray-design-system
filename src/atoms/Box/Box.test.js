import React from "react";

import { render } from "@testing-library/react";
import Box from "./Box";

test("it renders correctlyy", () => {
  const { container } = render(<Box />);
  expect(container.firstChild).toMatchSnapshot();
});
