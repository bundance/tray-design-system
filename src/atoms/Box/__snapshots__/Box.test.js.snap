// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders correctlyy 1`] = `
<div
  class="sc-AxjAm dQHWtE"
/>
`;
